esql/track.json

```diff
@@ -1,6 +1,18 @@
 {% import "rally.helpers" as rally with context %}
 {% set n_indices = (n_indices | default(5)) %}
 {% set n_fields = (n_fields | default(500)) %}
+
+{% set p_snapshot_repo_name = (snapshot_repo_name | default("logging")) %}
+{% set p_snapshot_repo_type = (snapshot_repo_type | default("s3")) %}
+{% set p_snapshot_repo_settings = (snapshot_repo_settings | default({
+"bucket": snapshot_bucket | default("test-bucket"),
+"client": "default",
+"base_path": snapshot_base_path | default("observability/logging"),
```

Comment:
The failures seem to be due to this default.
What should I use to make the CI happy?

Reply:
Indeed. CI for public Rally tracks does not have access to these buckets. Please skip this track in [here](https://github.com/elastic/rally-tracks/blob/00ad259d60f1f19c544440b8bfdefd77f5e34bed/it/test_all_tracks_and_challenges.py#L23). You can use empty `skip_challenges` for that but I would change it to a map, similarly to `disable_assertions`, for better precision.